cabal-install/src/Distribution/Client/Dependency.hs

```diff
@@ -756,14 +753,8 @@ standardInstallPolicy installedPkgIndex sourcePkgDb pkgSpecifiers =
 
 -- ------------------------------------------------------------
 
-chooseSolver :: Verbosity -> PreSolver -> CompilerInfo -> IO Solver
-chooseSolver _verbosity preSolver _cinfo =
-  case preSolver of
-    AlwaysModular -> do
-      return Modular
-
-runSolver :: Solver -> SolverConfig -> DependencyResolver UnresolvedPkgLoc
-runSolver Modular = modularResolver
+runSolver :: SolverConfig -> DependencyResolver UnresolvedPkgLoc
+runSolver = modularResolver
```

Comment:
Can we rename `modularSolver` to `runSolver` in `Distribution.Solver.Module`? I see this line as a left-over.